src/quacc/recipes/torchsim/core.py

```diff
@@ -0,0 +1,336 @@
+"""Core recipes for TorchSim."""
+
+from __future__ import annotations
+
+from importlib.util import find_spec
+from typing import TYPE_CHECKING, Any
+
+from monty.dev import requires
+
+from quacc import __version__, job
+from quacc.recipes.torchsim._base import (
+    pick_model,
+    process_binning_autobatcher_dict,
+    process_in_flight_autobatcher_dict,
+    process_trajectory_reporter_dict,
+)
+from quacc.runners._base import BaseRunner
+from quacc.schemas.torchsim import CONVERGENCE_FN_REGISTRY, ConvergenceFn, TSModelType
+
+has_torchsim = bool(find_spec("torch_sim"))
+if has_torchsim:
+    import torch_sim as ts
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+    from ase.atoms import Atoms
+
+    if has_torchsim:
+        from torch_sim.integrators import Integrator
+        from torch_sim.optimizers import Optimizer
+
+    from quacc.recipes.torchsim._base import (
+        AutobatcherDict,
+        TorchSimIntegrateSchema,
+        TorchSimOptSchema,
+        TorchSimStaticSchema,
+        TrajectoryReporterDict,
+    )
+
+
+@job
+@requires(has_torchsim, "torch_sim is required for this function")
+def relax_job(
+    atoms: list[Atoms],
+    model_type: TSModelType,
+    model_path: str | Path,
```

Comment:
@orionarcher Do all models need the `model_path`? For instance, the fairchem models don't require a model path. You can pass `name_or_path="uma-s-1"`, [for instance](https://github.com/facebookresearch/fairchem/blob/a9287b37469a80f9286151d70ceb184f91dfad8e/src/fairchem/core/calculate/ase_calculator.py#L124). This makes me feel that the recipes should support this as well. Do you agree?

Reply:
@sihoonchoi thanks for testing.

>It took some time to identify the appropriate arguments, but this shouldn't be an issue for users familiar with TorchSim.

We should assume the end user has no knowledge of TorchSim. Is there anything that we can do on the quacc side (e.g. docstrings) where you think it would have made the experience easier? Any things you were particularly unsure about?

Reply:
Thanks for your input, @sihoonchoi. I must admit that I do not really understand the value proposition for using `StrEnum`s when we can use a `str` with a `Literal["energy", "force"]` type hint.